packages/react/src/chat/BaseChat/BaseChat.tsx

```diff
@@ -73,6 +73,13 @@ export interface BaseChatProps extends PaperProps {
   readonly onError?: (err: Error) => void;
 }
 
+/**
+ * BaseChat component for displaying and managing communications.
+ * Component filters communcations where property sent is undefined.
```

Comment:
```suggestion
 *
 * **NOTE: The component automatically filters `Communication` resources where the `sent` property is `undefined`.**
```

Reply:
I like to bold notes like these to draw attention when hovering over them in the IDE